### PR TITLE
Updated delete organization endpoint functionality and implemented tests

### PR DESCRIPTION
**Pull request by AbdulWahab3181**
## Describe your changes

https://github.com/stakwork/sphinx-tribes/assets/41957704/1e37270f-f1a9-4b97-9fa1-56b1fa80a31c

## Issue ticket number and link

Closes https://github.com/stakwork/sphinx-tribes/issues/1334

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
There are a couple of conflicting files:
db/structs.go
handlers/organization_test.go

**Comment by AbdulWahab3181** [comment]
> There are a couple of conflicting files: db/structs.go handlers/organization_test.go

Resolved

**Comment by ecurrencyhodler** [comment]
LGTM! Good job. Let's get a code review.

File: db/db.go [hunk 1]
```diff
@@ -1466,3 +1466,39 @@ func (db database) ChangeOrganizationDeleteStatus(org_uuid string, status bool)
 	})
 	return ms
 }
+
+func (db database) UpdateOrganizationForDeletion(uuid string) error {
+	updates := map[string]interface{}{
+		"website":     nil,
```
Comment (elraphty) [inline db/db.go:1472]: @AbdulWahab3181 change all `nil` to empty strings `""`

File: handlers/organization_test.go [hunk 2]
```diff
@@ -99,3 +101,163 @@ func TestUnitCreateOrEditOrganization(t *testing.T) {
...
+
+	t.Run("should set Website, Github, and Description to null", func(t *testing.T) {
+		orgUUID := "org-uuid"
+
+		updatedOrg := db.Organization{
+			Uuid:        orgUUID,
+			OwnerPubKey: "test-key",
+			Website:     nil,
+			Github:      nil,
+			Description: nil,
```
Comment (elraphty) [inline handlers/organization_test.go:250]: @AbdulWahab3181 change  all `nil` to empty strings `""`

Comment (AbdulWahab3181) [reply handlers/organization_test.go:250]: @elraphty But Our [ticket](https://github.com/stakwork/sphinx-tribes/issues/1334) scope is nill. Let me know if it still needs to be changed.

image

image

Comment (ecurrencyhodler) [reply handlers/organization_test.go:250]: Follow Raph's advice here @AbdulWahab3181

Comment (AbdulWahab3181) [reply handlers/organization_test.go]: Addressed

File: db/db.go (hunk 1 above)
Comment (AbdulWahab3181) [reply db/db.go]: Addressed

File: handlers/organization_test.go (hunk 2 above)
Comment (elraphty) [reply handlers/organization_test.go]: > @elraphty But Our [ticket](https://github.com/stakwork/sphinx-tribes/issues/1334) scope is nill. Let me know if it still needs to be changed.
> 
> image image

Yeah, but using nil may lead to GORM errors e.g

https://github.com/go-gorm/gorm/issues/6351
https://github.com/go-gorm/gorm/issues/3363

**Comment by ecurrencyhodler** [comment]
Nice work. Paid!

**Pull request merged**